[Feature request] Move preview to first actual difference

Now we can get:
```
E       AssertionError: assert equals failed
E         {                                {                               
E           'landing_page': {                'landing_page': {             
E             'hgw': {                         'hgw': {                    
E               'left_section': {                'left_section': {         
E                 'texts': {                       'texts': {              
E                   'action_button': 'Sta            'action_button': 'Sta 
E         rt',                             rt',                            ...
E         
E         ...Full output truncated (469 lines hidden), use '-vv' to show
```

It would be cool to be able to see actual difference in dicts in a preview. Output could be scrolled to first difference, e.g.:

```
E       AssertionError: assert equals failed
E         …                               …                               
E           'in_the_wild': {                'in_the_wild': {             
E             'sth': {                         'sth': {                    
E               'some_section': {                'some_section': {         
E                 'fooos': {                       'fooos': {              
E                   'difference': 'foo',            
E                                                    'difference': 'bar',                         …
E         
E         ...Full output truncated (469 lines hidden), use '-vv' to show
```

--- a/pytest_icdiff.py
+++ b/pytest_icdiff.py
@@ -52,6 +52,6 @@ def pytest_assertrepr_compare(config, op, left, right):
     else:
         color_off = icdiff.color_codes['none']
 
-    icdiff_lines = list(differ.make_table(pretty_left, pretty_right))
+    icdiff_lines = list(differ.make_table(pretty_left, pretty_right, context=True))
 
     return ['equals failed'] + [color_off + l for l in icdiff_lines]

